doc/cephadm/services/index.rst

```diff
@@ -587,6 +634,55 @@ the node-exporter service , one could apply a service spec like
   extra_entrypoint_args:
     - "--collector.textfile.directory=/var/lib/node_exporter/textfile_collector2"
 
+There are two ways to express arguments in the ``extra_entrypoint_args`` list.
+To start, an item in the list can be a string. When passing an argument as a
+string and the string contains spaces, cephadm will automatically split it into
+multiple arguments. For example, ``--debug_mon 10`` would become
+``["--debug_mon", "10"]`` when processed. Example:
```

Comment:
it doesn't really matter since it's just an example arg, but was this meant to be `debug_ms`?

Reply:
heh, yes. I was looking for a sufficiently real example and was reading another help doc to see what config options I could pass to a mon, but I must have got my mental wires crossed.

Reply:
Fixed.